extensions/wikia/ArticleComments/modules/templates/ArticleComments_Comment.php

```diff
@@ -12,7 +12,7 @@
 
 		<div class="edited-by">
 		<?= wfMessage( 'oasis-comments-added-by' )->rawParams( $comment['timestamp'], $comment['sig'] )->escaped() ?>
-		<?php if (!empty($comment['isStaff'])) { print "<span class=\"stafflogo\"><img src=\"".wfReplaceImageServer( '/extensions/wikia/StaffSig/images/WikiaStaff.png' ) . "\" title=\"This user is a member of Wikia staff\" alt=\"@wikia\" /></span>\n"; } ?>
+		<?php if (!empty($comment['isStaff'])) { print "<span class=\"stafflogo\"><img src=\"".wfReplaceImageServer( wfGetSignatureUrl() ) . "\" title=\"This user is a member of Fandom staff\" alt=\"@fandom\" /></span>\n"; } ?>
```

Comment:
What if we used `DesignSystemHelper::getSvg` helper function here and everywhere we need the new logo instead of manually generating our HTML? This seems to be the preferred method of using Design System assets.